crates/uv-auth/src/middleware.rs

```diff
@@ -301,6 +301,13 @@ impl AuthMiddleware {
     /// Run a request to completion.
     ///
     /// If credentials are present, insert them into the cache on success.
+
+    fn fetch_env_credentials(&self, url: &Url) -> Option<Credentials> {
+        let host = url.host_str()?;
+        let index_name = host.replace('.', "_").to_uppercase();
+        Credentials::from_env(&index_name)
```

Comment:
I don't think this is quite right -- the index name is not necessarily the same as the URL host, it's a user-provided string.

Reply:
How can I get the `name` from the pyproject.toml file.

Reply:
@charliermarsh , 
Can you help me by telling me the position of getting index_name by the url?